pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -724,7 +717,7 @@ func checkMetadataSearchOrder(order string) error {
 		case configDriveID:
 		case metadataID:
 		default:
-			errTxt := "Invalid element '%s' found in section [Metadata] with key `search-order`." +
+			errTxt := "invalid element '%s' found in section [Metadata] with key `search-order`." +
```

Comment:
Consider using `%q` rather than `'%s'`

Reply:
Also: move this string format into the `fmt.Errorf` call - so linters are more likely to be able to recognise the format specifiers and match them with args.